### DISCUSSION TITLE
Access Hubble UI from local network

### DISCUSSION THREAD

**Discussion by rubiktubik** [onedr0p/cluster-template#1451] [Q&A] [answered]
Thanks you for great template @onedr0p !

I have successfully setup my kubernetes cluster with talos but cannot reach the hubble-ui from my local network.

I added this to my Local-DNS-Server:
[/${bootstrap_cloudflare.domain}/]${bootstrap_cloudflare.gateway_vip}

When using dig it get the ip of the internal_ingress_route IP. It seems to work!

When executing:
`task kubernetes:resources`

the ingress for internal hubble-ui is also correct

But in the browser i get a https certificate problem.
<img width="277" alt="grafik" src="https://github.com/onedr0p/cluster-template/assets/115047/518cb592-46d4-4ec1-8ac8-81903d2af20a">

Does the hubble-ui route has no certificate and https configured?

I find no annotations in the yaml files

**Comment by onedr0p** [comment, marked as answer]
Did you switch over to the LE production servers? It's detailed in the README.

**Reply by rubiktubik** [reply]
Switching to LE prod-certificates was the trick. Now it works! 
Thank you for the hint.

But out of interest, where is it defined that an Ingress by default use the wildcard certificate?

Is it this line:

```yaml
extraArgs:
        default-ssl-certificate: "network/${SECRET_DOMAIN/./-}-production-tls"
```

In this file:
![grafik](https://github.com/onedr0p/cluster-template/assets/115047/c6ad0a35-75a4-4fec-a49a-17b787d941ce)

If I don't want to work with wildcart certificates but want to create a certificate on demand for a specific route? Then I have to make an annotation like in the Certmanager documentation, right? Or Nginx documentation

**Reply by onedr0p** [reply]
Yes on all fronts 😁
